packages/cms-editor/src/components/FileValueField.tsx

```diff
@@ -50,15 +49,11 @@ export const FileValueField = ({ updateValue, deleteValue, setFileExtension, all
           disabled={baseProps.disabled}
           ref={inputRef}
         />
-        {allowOpenFile && baseProps.contentObject.values[baseProps.language.value] && (
+        {allowOpenFile && value && (
           <TooltipProvider>
             <Tooltip>
               <TooltipTrigger asChild>
-                <Button
-                  icon={IvyIcons.File}
-                  aria-label={t('value.openFile')}
-                  onClick={() => openUrl(`${cmUrl}${baseProps.contentObject.uri}?l=${baseProps.language.value}`)}
-                />
+                <Button icon={IvyIcons.File} aria-label={t('value.openFile')} onClick={() => openUrl(value as string)} />
```

Comment:
is the value an url or just the content object path? 
Because if the value is an URL, maybe value is not a good name for it...